Makefile

```diff
@@ -1,5 +1,13 @@
-build: 
-	@cdk synth
+install:
+	@corepack enable
+	@yarn set version stable
```

Comment:
If we localised (frozen) it well, we may rearrange `corepack enable` and `yarn set version stable` to another Makefile target (such as yarn-deps-bump) as we may not want to pull in latest yarn with every other CI build. Instead we inform CI environment (and other developer local dev) to leverage the localised yarn version from the repo.

We run these `yarn set version stable` in another time when we wish to bump yarn version itself in controlled dependencies upgrade manner.

Reply:
Right, so I think the set yarn version is pinned based on the `packageManager` field on the `package.json` file. It seems that this is the new recommended way of installing yarn rather than the local binary checked in the to repo from the v4 installation guide: https://yarnpkg.com/blog/release/4.0#installing-yarn

But do agree that the install target should only be install commands instead of any env setup, so this could move to another makefile target.